frontend/pages/passport-application/index.tsx

```diff
@@ -0,0 +1,99 @@
+import Link from "next/link"
+
+import Logo from "../../shared-components/logo"
+import { USStateSelect } from "../../shared-components/state-select/UsStateSelect"
+import styles from "./passport.module.css"
+
+function handleSubmit() {
+  // TODO: add form submission logic
+}
+
+export default function Passport({ name = "Herbert Q. Placeholder" }) {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.logowrapper}>
+        <Logo />
+      </div>
+      <h1 className={styles.heading}>Passport Account Application</h1>
+      <div className={styles.paraWrapper}>
+        <p className={styles.para}>
+          Hello <em>{name}</em>, thank you for your continued interest in the National Police Data
+          Coalition.
+        </p>
+        <p className={styles.para}>
+          We are able to provide access to legally protected data to users with the appropriate
+          permissions. This form will submit your profile for approval.
+        </p>
+      </div>
+      <form className={styles.form}>
+        <div className={[styles.address, styles.inputwrapper].join(" ")}>
+          <label htmlFor="address" className={styles.label}>
+            <strong>Street Address:</strong>
+          </label>
+          <input
+            type="text"
+            id="address"
+            placeholder="Enter your street address"
```

Comment:
Hey! So thanks for removing the Bob Boberton stuff 😅 but the wireframes actually don't have any placeholder text at all & that's intentional as it's considered inaccessible/bad UX

Here's an article I found helpful on why that is! 
https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/